docs/application/native/guides/text-input/autofill-service.md

```diff
@@ -0,0 +1,328 @@
+# Autofill Service
+
+Autofill Service application is a service application to store the user input and provide the data previously entered by the user.
+Using the autofill service functions, Autofill service application developers can implement their own autofill service to serve the autofill information.
+
+The main features of the Autofill service API include:
+
+- [Receiving the request of authentication information and sending the authentication information](#receiving-the-request-of-authentication-information-and-sending-the-authentication-information)
```

Comment:
Receiving request of authentication information and sending authentication information